utils/src/server/rpc_module.rs

```diff
@@ -199,6 +228,33 @@ impl Methods {
 		rx.next().await
 	}
 
+	/// Test helper that sets up a subscription using the given `method`. Returns a tuple of the
+	/// [`SubscriptionId`] and a channel on which subscription JSON payloads can be received.
+	pub async fn test_subscription(
+		&self,
+		method: &str,
+		params: Option<Box<RawValue>>,
+	) -> (SubscriptionId, mpsc::UnboundedReceiver<String>) {
+		log::trace!("[Methods::test_subscription] Calling subscription method: {:?}, params: {:?}", method, params);
+		let req = JsonRpcRequest {
+			jsonrpc: TwoPointZero,
+			id: Id::Number(0),
+			method: Cow::borrowed(method),
+			params: params.as_deref(),
+		};
+
+		let (tx, mut rx) = mpsc::unbounded();
+
+		if let Some(fut) = self.execute(&tx, req, 0) {
+			fut.await;
+		}
+		let response = rx.next().await.expect("Could not establish subscription.");
+		let subscription_response = serde_json::from_str::<JsonRpcResponse<SubscriptionId>>(&response)
+			.expect(&format!("Could not deserialize subscription response {:?}", response));
```

Comment:
```suggestion
			unwrap_or_else(|| panic!("Could not deserialize subscription response {:?}", response));
```

Reply:
Why is `unwrap_or_else` + `panic` better here?